### PR TITLE
0.22 concent mainnet

### PR DESCRIPTION
**Pull request by shadeofblue**

### CODE REVIEW

**Review by maaktweluit — APPROVED** [review]
LGTM!

Did not check the terms ;)

File: golem/CONCENT_TERMS.html [hunk 1]
```diff
@@ -1,7 +1,371 @@
-<h5><u>Disclaimer</u></h5>
+<h4><u>Terms of Use of Concent Service</u></h4>
+<p><em>Last Update: June 2019.</em></p>
+<p>
+    These Terms of Use of Concent Service (hereinafter the “Terms of Use”)
```
Comment (etam) [inline golem/CONCENT_TERMS.html]: I'd make sure this is a simple ascii text file, not utf8. So all `“` should be changed to `"`.

Comment (etam) [reply golem/CONCENT_TERMS.html]: and `’` to `'`

Comment (shadeofblue) [reply golem/CONCENT_TERMS.html]: yup, detected by the test suite and fixed already :)

**Pull request merged**